MatrixSDK/Crypto/Verification/MXKeyVerificationManager.h

```diff
@@ -85,12 +85,6 @@ FOUNDATION_EXPORT NSString *const MXKeyVerificationManagerNotificationTransactio
 
 #pragma mark - Requests
 
-/**
- The timeout for requests.
- Default is 5 min.
- */
-@property (nonatomic) NSTimeInterval requestTimeout;
```

Comment:
`requestTimeout` and `keyVerificationIdFromDMEvent` is not in fact called from outside of the manager, so can be hidden and not overriden by manager v2